test/Dashboard.spec.js

```diff
@@ -8,16 +9,18 @@ import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
 // local
 import dashboard from '@/pages/dashboard.vue'
 import UserProduct from '@/components/UserProduct.vue'
-import { state } from '@/store'
+import { state, actions, getters, mutations } from '@/store'
 import { ProductCode, ProductStatus } from '@/enums'
 import { authAxios, getProductInfo } from '@/utils'
 import { createComponent } from '@/test/utils'
 
 Vue.use(Vuetify)
+Vue.use(Vuex)
 const vuetify = new Vuetify({})
-const store = state
+const store = new Store({ state, getters, mutations, actions })
```

Comment:
This is inconsistent with the use of `store` in our other UIs, where [store/index.ts](https://github.com/bcgov/business-edit-ui/blob/main/src/store/index.ts) creates the store, and both the app and the tests can use it.

Does this app create the store elsewhere, and you can't tap into that here?

Reply:
For this app, we don't have the store set up this way, so I added it to the test.

Reply:
Thanks, that would be great.

Reply:
I'm assuming the store works currently in the app, though I'm not sure how it's instantiated. Maybe you cannot tap into that mechanism in the tests; I dunno. If you can't find it then don't worry about it.

Reply:
Updated to use globally defined VuexStore